### PR TITLE
feat: Added drizzle scaffolding

### PR DESCRIPTION
**Pull request by gregmulvaney**
PR adding basic drizzle scaffolding for Issue #10

### CODE REVIEW

File: .env.example
Comment (phillip-hirsch) [inline .env.example:1]: **issue:** We should find a more secure solution to deal with secrets instead of using `.env` files

Comment (gregmulvaney) [reply .env.example:1]: .env files are merely for development. Node's `process.env` also reads from system ENV vars.  Deployment would ideally use actual system ENV vars (e.g. [Docker container ENV vars](https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/)) and database security features such as Postgres' [RLS](https://www.postgresql.org/docs/current/ddl-rowsecurity.html#DDL-ROWSECURITY) to protect access to data.

Comment (phillip-hirsch) [reply .env.example:1]: Ok, got it!

File: src/db/index.ts
[the remark this answers is not in the document]
Comment (phillip-hirsch) [reply src/db/index.ts:1]: That makes sense

**Review by phillip-hirsch — APPROVED** [review]
LGTM

**Pull request merged**